### PR TITLE
Adding exception KeyError and changing currency_choices

### PR DESCRIPTION
**Pull request by JuanVT**
-Adding exception KeyError. In one case particularly when unit and unit_to equals 'EUR' these can't be found in the API which ends in a KeyError.

-Changing currency_choices for better readability in front end.

### CODE REVIEW

File: control/forms.py [hunk 1]
```diff
@@ -64,39 +64,39 @@ class TimeConverter(forms.Form):
...
+    ('cny', 'Chinese Yuan'),
+    ('nok', 'Norwegian Krone'),
+    ('nzd', 'New Zealand Dollar'),
+    ('zar', 'South African Rand'),
+    ('usd', 'United States Dollar'),
+    ('mxn', 'Mexican Peso'),
+    ('ils', 'Israeli New Shekel'),
+    ('gbp', 'Pound sterling'),
+    ('krw', 'South Korean Won'),
+    ('myr', 'Malaysian Ringgit')
```
Comment (TysonRV) [inline control/forms.py]: This belongs to a different pull request

Comment (TysonRV) [reply control/forms.py]: Hardcoding is never a good practice anyway, I would suggest putting this in a model, in a database. You might be able to use an existing one and store them there, look at: https://github.com/panosl/django-currencies/

File: control/views.py [hunk 2]
```diff
@@ -144,12 +144,15 @@ def currency_converter(request):
...
-            url = 'https://api.exchangeratesapi.io/latest?base='+unit
-            currency_values = requests.get(url).json()
-            currency_value = currency_values['rates'][unit_to]
-
-            calculations = unit_value * currency_value
-            result = '{} {}'.format(calculations, unit_to)
+            try:
+                url = 'https://api.exchangeratesapi.io/latest?base='+unit
+                currency_values = requests.get(url).json()
+                currency_value = currency_values['rates'][unit_to]
```
Comment (TysonRV) [inline control/views.py:151]: When `try...except..` an error, you have to be as much specific as you can. In this case, this error is only affecting the line: `currency_value = currency_values['rates'][unit_to]`  so...

```suggestion
            url = 'https://api.exchangeratesapi.io/latest?base='+unit
            currency_values = requests.get(url).json()
        try:
            currency_value = currency_values['rates'][unit_to]
        except KeyError:
            currency_value = 1
        else:
            calculations = unit_value * currency_value
            result = '{} {}'.format(calculations, unit_to)
```

Comment (JuanVT) [reply control/views.py:151]: Ok, I get it it's more clean this way, what I don't really understand is the else: on line 156, that makes the code ignore the formula and it will always return 0, even if you try to convert 1 euro to eur, it should return 1 not 0. I've omited the else and with the right indent works fine I'll push it so you can see it.

File: control/forms.py (hunk 1 above)
Comment (JuanVT) [reply control/forms.py]: ok, I wasn't really sure if this was hard code or not I'll have a look, I've pushed it in the same PR because it was related to currency_converter, but I can always open a different branch.

File: control/views.py (hunk 2 above)
Comment (TysonRV) [reply control/views.py:151]: Yes, it shouldn't be an `else` clause, it should be `finally` instead. Good shot

Comment (JuanVT) [reply control/views.py:151]: it should? or it shouldn't? haha you're confusing me

Comment (TysonRV) [reply control/views.py:151]: lol shouldn't
